client/src/components/Overview.jsx

```diff
@@ -26,24 +26,23 @@ class Overview extends React.Component {
     this.fetchRestaurantInfo();
   }
 
-  fetchRestaurantInfo() {
-    // const id = window.location.href.split('/')[4];
-    let id = Math.floor(Math.random() * Math.floor(1000000));
-    console.log('id', id);
+  fetchRestaurantInfo () {
+    // let id = Math.floor(Math.random() * Math.floor(1000000));
+    let context = this;
+    let id = window.location.href.split('/')[4];
+    console.log('IM ID......',id);
 
```

Comment:
I would remove some of the console.log notes to self, if possible - if you need to keep them for tracking a particular feature change, I would snag them and save them on a new feature branch.